onpremise/metaissue.go

```diff
@@ -78,6 +131,57 @@ func (s *IssueService) GetCreateMeta(ctx context.Context, options *GetQueryOptio
 	return meta, resp, nil
 }
 
+func (s *IssueService) GetCreateMetaProjectIssueTypes(ctx context.Context, projectKey string, options *GetQueryIssueTypeOptions) ([]*MetaIssueType, *Response, error) {
+	apiEndpoint := fmt.Sprintf("rest/api/2/issue/createmeta/%s/issuetypes", projectKey)
+
+	req, err := s.client.NewRequest(ctx, http.MethodGet, apiEndpoint, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	if options != nil {
+		q, err := query.Values(options)
+		if err != nil {
+			return nil, nil, err
+		}
+		req.URL.RawQuery = q.Encode()
+	}
+
+	meta := new(CreateMetaProjectIssueTypeInfo)
+	resp, err := s.client.Do(req, meta)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return meta.Values, resp, nil
```

Comment:
I don't know how this will be consumed, but I think it'll be helpful to have the `IsLast` field when looping and paginating. Up to you though.
```go
return meta, resp, nil
```

Reply:
good point ty https://github.com/ConductorOne/go-jira/pull/8/commits/3fadcdbe9a724604e6ee3b5e08d1b51e361a58e1